mm2src/adex_cli/src/scenarios/init_mm2_cfg.rs

```diff
@@ -51,7 +51,7 @@ struct Mm2Cfg {
     #[serde(skip_serializing_if = "Vec::<Ipv4Addr>::is_empty")]
     seednodes: Vec<Ipv4Addr>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    hd_account_id: Option<u64>,
+    enable_hd: Option<bool>,
```

Comment:
I guess it could be bool initialized by default

Reply:
P.S.: It is also could have a sense to introduce an enum instead of boolean to express an addressing scheme

Reply:
> I guess it could be bool initialized by default

I followed what you did with other parameters, e.g.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/2346095a8272a211c2d2325117166ba21db35a0a/mm2src/adex_cli/src/scenarios/init_mm2_cfg.rs#L49-L50
should I change others too?

Reply:
No, thank you! It's ok if `enable_hd` could be missed in the configuration.